### ISSUE TITLE
[BUG] Setting cursor-color has no impact on the Rofi application launcher.

### ISSUE THREAD

**Issue by mpieczaba** [davatorium/rofi#1981]
### Rofi version (rofi -v)

Version: 1.7.5

### Configuration

https://gist.github.com/mpieczaba/3a1a1220c1533385ce2a26048d208a68

### Theme

https://gist.github.com/mpieczaba/3669d07560a7bf41907af32be9e788f6

### Launch command

rofi -show combi -combi-modes 'window,drun,run'

### Step 1

Set theme to the one in the attached gist.

### Step 2

Try to type something. 

### Expected behavior

Text cursor color should have been changed to yellow.

### Actual behavior

Cursor is white, not yellow-ish.

### Using wayland display server protocol

- [X] No, I don't use the wayland display server protocol

### I've checked if the issue exists in the latest stable release

- [ ] Yes, I have checked the problem exists in the latest stable version

**Comment by DaveDavenport** [comment]
Thanks for the report:

```bash
rofi  -show drun -theme-str 'entry { cursor-color: pink; }'
```

This works for me in git.

**Comment by DaveDavenport** [comment]
This is not a feature available in 1.7.5. Not a bug.

**Comment by DaveDavenport** [comment]
See https://davatorium.github.io/rofi/1.7.5/rofi-theme.5/

**Comment by mpieczaba** [comment]
> This is not a feature available in 1.7.5. Not a bug.

Yeah, sorry. I haven't checked the docs version. Thank you for your quick response and clearing this out.
